src/WorldSelector.scss

```diff
@@ -0,0 +1,40 @@
+@import 'variables';
+
+.WorldSelector {
+    background-color: #CCE6E2;
+    padding: 1rem;
+    border: 1px solid grey;
+    border-radius: 3px;
+    height: 100%;
+}
+
+.WorldSelector .HeaderIcon, .WorldSelector .WorldIcon {
+    display: block;
+    height: 4rem;
+    width: 4rem;
+    padding: 0.2rem;
+}
+
+.WorldSelector .HeaderIcon {
+    background: none;
+}
+
+.WorldSelector .WorldIcon {
+    box-shadow: inset 0 .1rem .2rem rgba(0, 0, 0, 0.25);
+    background-color: white;
+    border-radius: 5px;
+    margin-top: 0.8rem;
+    padding: 0.4rem;
+}
+
+.WorldSelector .WorldIcon--selected {
+    background: #88C3BC;
+}
+
+.WorldSelector button:focus {
```

Comment:
Focus and hover styles are applied to button component, but these weren't applying because current WorldSelector is not a button.

Reply:
I see them now.